src/components/Tooltip/index.tsx

```diff
@@ -14,11 +15,35 @@ export interface IProps extends PopoverProps {
 }
 
 const Tooltip: React.FC<IProps> = ({ content, children, ...rest }) => {
+  const { isOpen, onOpen, onClose } = useDisclosure()
+
+  // Close the popover when the user scrolls.
+  // This is useful for mobile devices where the popover is open by clicking the
+  // trigger, not hovering.
+  useEffect(() => {
+    const handleScroll = () => {
+      if (isOpen) {
+        onClose()
+      }
```

Comment:
Consider a guard clause

```suggestion
      if (!isOpen) return
      onClose()
```

...or we could consider removing the check all together, but I think this is good